From 0b8c7715ea253f617578ee1cdcbd4398a91c1959 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Thu, 18 Dec 2014 11:58:57 -0500
Subject: [redhat] kabi: Adding symbol __ndelay (arch/x86/um/delay.c)

Message-id: <dc9d1b64fc44977501abb54cb054bebe44ba9693.1418903403.git.poros@redhat.com>
Patchwork-id: 102237
O-Subject: [PATCH RHEL7.1 05/48] kabi: Adding symbol __ndelay (arch/x86/um/delay.c)
Bugzilla: 1160762
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1160762
    https://bugzilla.redhat.com/show_bug.cgi?id=1160762

    Description:
    ============
    Called indirectly via ndelay, delay CPU to allow fpga loat load new values.
    Used in pcie_rt.ko

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/__ndelay
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/__ndelay

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_s390x/__ndelay b/redhat/kabi/kabi-module/kabi_s390x/__ndelay
new file mode 100644
index 0000000..f370361
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/__ndelay
@@ -0,0 +1,2 @@
+#1-
+0x338bbef8 __ndelay vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/__ndelay b/redhat/kabi/kabi-module/kabi_x86_64/__ndelay
new file mode 100644
index 0000000..374cc88
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/__ndelay
@@ -0,0 +1,2 @@
+#1-
+0xdf8c695a __ndelay vmlinux EXPORT_SYMBOL
-- 
1.7.1